Python3 updates

slremy opened the pull request:
Hello!
Some things took more time to track down than others, but please take a look at changes that I've suggested when using with python3.  Some of the changes are coming from the execution of the 2to3 tools, but most are from encountering things that changed when trying to run our old code.

Note: I didn't touch https://github.com/SwissTPH/openmalaria.tools/tree/master/bin since there was an April 2016 commit that mentioned Py3 conversion.

Commit:
updating a few lingering print statements

slremy committed:
Merge pull request #1 from slremy/Python3

updating a few lingering print statements

slremy committed:
Python3 (#2)

* application of 2to3

* since string method is deprecated, removing it

* making plotting better

* making plotting better

* updating shebangs to python3

* updating to optionally read from StringIO object

* these appear to refer to integer division

* adding BytesIO

dhardy commented:
So this supercedes the other PR?

The plotResult script in bin should be updated to python3.

slremy commented:
Yes, this would supercede the previous PR.

slremy committed:
removing explicit python2 reference

With my testing this call works. On my machine, as is, since I have both python2 and python3 installed it runs the python2 interpreter. To make it use python3, line 2 should be changed to python3 -m openmalaria.tools.plotResult $@

slremy committed:
Updating per discussion in PR
